src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -179,7 +168,7 @@ function BaseSelectionList({
             // Otherwise, it will cause an index-out-of-bounds error and crash the app.
             let adjustedSectionIndex = sectionIndex;
             for (let i = 0; i < sectionIndex; i++) {
-                if (_.isEmpty(lodashGet(sections, `[${i}].data`))) {
+                if (sections[i].data) {
```

Comment:
Is optional chain needed here?
```suggestion
                if (sections?.[i]?.data) {
```

Reply:
No need - sections is required